Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.tracers

```diff
@@ -0,0 +1,44 @@
+# ------------------  INPUTS TO MAIN PROGRAM  -------------------
```

Comment:
Similarly, this inputs file is not needed by this problem - it pertains to tracers, but tracer particles are not currently available in the Fortran interfaces.

Reply:
You can close this one. Opening a new one with newly created fork.